s3sync.sh

```diff
@@ -3,12 +3,16 @@
 check_command s3cmd
 check_command awk
 
-: ${AWS_ACCESS_KEY:?"AWS Access key not specified (AWS_ACCESS_KEY)"}
-: ${AWS_SECRET_KEY:?"AWS Secret key not specified (AWS_SECRET_KEY)"}
-: ${AWS_S3_BUCKET:?"AWS Bucket name not specified (AWS_S3_BUCKET)"}
+: ${AWS_ACCESS_KEY:=}
+: ${AWS_SECRET_KEY:=}
+: ${AWS_S3_BUCKET:=}
 : ${AWS_S3_BUCKET_PATH:=}
 
 s3sync() {
+    is_enabled() {
+        [ -n "${AWS_ACCESS_KEY}${AWS_SECRET_KEY}" ]
```

Comment:
This will return true, indicating that S3 sync is enabled, even if not both access key and secret key are set, which would be invalid.